docs/fastdds/dds_layer/topic/contentFilteredTopic/createContentFilteredTopic.rst

```diff
@@ -26,9 +28,15 @@ Mandatory arguments are:
 Optional arguments are:
 
  * A string with the name of the filter class to use for the filter creation.
-   This allows the user to create filters different from the standard SQL like one.
+   This allows the user to create filters different from the standard SQL like one
+   (please refer to :ref:`dds_layer_topic_contentFilteredTopic_custom_filters`).
    Defaults to ``FASTDDS_SQLFILTER_NAME``.
```

Comment:
Either we put the correct name or we add this to the API reference

Reply:
Added to the API reference